setup.py

```diff
@@ -3,7 +3,7 @@
 setup_reqs = ['pytest', 'pytest-cov', 'pytest-runner', 'flake8']
 setuptools.setup(
     name="gordian",
-    version="2.1.2",
+    version="2.2.0",
```

Comment:
The default behavior now is to not fork right? this should be a major change

Reply:
Yep, good point...